app/src/main/java/com/bashkevich/androidfundamentals/model/dto/ActorDto.kt

```diff
@@ -0,0 +1,22 @@
+package com.bashkevich.androidfundamentals.model.dto
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CreditsResponse(
+    @SerialName("id")
```

Comment:
Why to have it if property name is the same?

Reply:
In was in lecture when he said that sometimes we can  occasionally change the field value